### PR TITLE
Improve the README

### PR DESCRIPTION
**Pull request by ryb-ableton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,21 +92,23 @@ This software is distributed under the [MIT License](./LICENSE).
 
 ⚠️ Drawing visualizations is expensive and can impact the scheduling of audio threads in surprising ways. You can observe behavior without the confounding effect of drawing by briefly turning visualizations off, performing a test, and then turning them back on.
 
-## Load
+## Visualizations
+
+### Load
```
Comment (aza-ableton) [inline README.md:97]: Now that there is much more text above this section you have to scroll to actually see the UI element each paragraph refers to. Some of the UI elements, such as "Work distribution" are collapsed in the screen shot, so you don't see what is being talked about until you build and run the app.

Might be nice to have a picture of each UI element before each paragraph. You'd then get a better idea about what the app does before downloading and building it.

**Review by aza-ableton — COMMENTED** [review]
Nice! I couldn't find any errors, but I'd have a "usability" remark.

File: README.md (hunk 1 above)
Comment (ryb-ableton) [reply README.md:97]: I agree and have thought about this, but I don't want to have to chop up images to change the UI, so I decided against it . Updating the single screen screen is already enough work.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,28 @@
...
+
+The [cores visualization](#cores) shows how audio threads are scheduled onto cores over time and the [energy visualization](#energy) shows power consumption. The main two workarounds to avoid CPU throttling are adding [artificial load to real-time threads](#minimum-load) and running [busy threads](#busy-threads) that perform low-energy work in the background. There are additional controls to tweak things like the [number of audio threads](#process-threads) and if those threads [join a workgroup](#work-interval).
+
+See [High Performance Audio on iOS](https://youtu.be/ywrLAv5WNq4) from ADC '19 for more information.
+
+# Recommendations
+
+On iOS 13, real-time multi-threading doesn't work at low buffer sizes (< 512). For these buffer sizes, worker threads get scheduled onto efficiency cores with low clock speeds. These threads run so slow that a task that could normally (on a fast core) be processed in 0.5ms ends up taking more than 3ms, overshooting the 2.7ms deadline for a buffer size of 128.
```
Comment (drd-ableton) [inline README.md:47]: Nit: "a task that could be processed in 0.5ms on a fast core ends up..." might read a bit better, the "normally" with a parenthetical is a bit awkward.

**Review by drd-ableton — APPROVED** [review]
Nice.  Reads well to me.  I left one minor wording suggestion.

File: README.md (hunk 2 above)
Comment (ryb-ableton) [reply README.md]: Good point, changed!

File: README.md (hunk 1 above)
Comment (aza-ableton) [reply README.md:97]: Fair enough.

**Pull request merged**